chart/kubeapps/values.yaml

```diff
@@ -335,7 +335,7 @@ hooks:
   image:
     registry: docker.io
     repository: bitnami/kubectl
-    tag: 1.19.9-debian-10-r7
+    tag: 1.19.10-debian-10-r9
```

Comment:
Since kubectl 1.21 has been released, kubectl 1.19 will be deprecated sooner or later, perhaps we should think of changing it someday.

Reply:
Yep, let's do it straight after this release in case there's any deprecations is 1.21 apis that cause us pain.